index.html

```diff
@@ -39,8 +39,9 @@ <h1 class="green heading subheading">I'm a software developer</h1>
   <!-- Works setion  -->
   <section id="works">
```

Comment:
- [ ] Please the works section should have the same margins (top, bottom, left, right) defined in the design there is a weird distance on the right of the work section make sure to fix   it I think it is because of the headline section
![image](https://user-images.githubusercontent.com/74744787/235235656-fd34cc92-8979-483f-850b-9d3dd9876b0b.png)
Figma design
![image](https://user-images.githubusercontent.com/74744787/235236178-37ec5371-8b3a-4f96-9bff-3a09ddf2935f.png)